ccd-definition/CaseEventToFields/AddDefendantLitigationFriend.json

```diff
@@ -2,11 +2,34 @@
   {
     "CaseTypeID": "CIVIL",
     "CaseEventID": "ADD_DEFENDANT_LITIGATION_FRIEND",
-    "CaseFieldID": "respondent1LitigationFriend",
```

Comment:
If we are no longer using the `respondent1LitigationFriend` field, this either needs to be
A) New changes toggled off behind a non-prod file or
B) We hard swap to use: `genericLitigationFriend` + the callback will populate `respondent1LitigationFriend` with a copy. 
Currently your callback is saying: if multiparty is not enabled, use existing callback. And the exiting callback, only sets the litigation friend date and created date